examples/static-site-gen/platform/src/lib.rs

```diff
@@ -313,3 +327,10 @@ fn is_roc_code_block(cbk: &pulldown_cmark::CodeBlockKind) -> bool {
         }
     }
 }
+
+fn replace_code_with_static_file(code : &str, input_dir: &Path) -> Option<String> {
+    let file_path = input_dir.join(code.trim());
+    let vec_u8 = fs::read(file_path).ok()?;
+
+    String::from_utf8(vec_u8).ok()
+}
```

Comment:
I now understand what you meant with your performance comment.
I don't think we should try to open a file with whatever code is in the block.
I think we should verify if the trimmed code is a single line and contains at least one `.` before trying to read it.